### PR TITLE
Payment profile setting Page construction

### PR DESCRIPTION
**Pull request by DhaaraniCIT**
## Summary by CodeRabbit

- **New Features**
  - Added support for TravelPerk integration, including payment profile settings and advanced configuration options.
- **Enhancements**
  - Improved user interface for payment profile configuration with conditional rendering for better user experience.
  - Updated user interface elements and styling for configuration mapping fields, enhancing visual clarity and usability.
- **Bug Fixes**
  - Fixed an issue with incorrect property naming (`update_at` to `updated_at`) in TravelPerk payment profile settings.
  - Corrected dropdown behavior and error message displays in configuration mapping fields for a more intuitive user interaction.

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
@coderabbitai review

**Comment by DhaaraniCIT** [comment]
Payment Profile Settings
Screenshot 2024-02-02 at 3 27 39 PM

File: src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.html [hunk 1]
```diff
@@ -1 +1,69 @@
...
+                        </h4>
+                        <p class="tw-text-14-px tw-text-faded-text-color tw-w-600-px tw-text-prrety">
+                            Add the required Payment Profiles and assign a User Role to each of them based on your requirement. 
+                            Expenses will be created for every invoice generated in Travelperk based on the settings.
+                            <a class="tw-text-hyperlink-color tw-pl-2-px tw-w-fit tw-cursor-pointer" (click)="showPreviewDialog(true)">
+                                Click here
+                            </a> 
+                            to preview.
+                        </p>
+                    </div>
```
Comment (ashwin1111) [inline src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.html:29]: don't we have some common component for this?

File: src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.html [hunk 2]
```diff
@@ -1 +1,69 @@
...
+                    (navigateToPreviousStep)="navigateBack()"
+                    (save)="save()">
+                </app-configuration-step-footer>
+            </div>
+        </form>
+    </div>
+    <div *ngIf="isPreviewDialogVisible">
+        <app-preview-dialog 
+            [isPreviewDialogVisible]="true"
+            [iconPath]="'assets/illustrations/sageIntacct/IntacctImportSettings.png'" (closeDialog)="closeDialog()"
```
Comment (ashwin1111) [inline src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.html:65]: add a todo to update this

File: src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.ts [hunk 3]
```diff
@@ -33,6 +36,12 @@ export class TravelperkPaymentProfileSettingsComponent implements OnInit {
 
   sessionStartTime: Date = new Date();
 
+  supportArticleLink: string = brandingKbArticles.onboardingArticles.SAGE300.IMPORT_SETTING;
```
Comment (ashwin1111) [inline src/app/integrations/travelperk/travelperk-shared/travelperk-payment-profile-settings/travelperk-payment-profile-settings.component.ts:39]: travelperk article

**Review by ashwin1111 — APPROVED** [review]

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**